src/gitter.rs

```diff
@@ -47,7 +47,7 @@ impl<'a> Gitter<'a> {
 
     /// Returns the current user
     pub fn get_user(&self) -> ApiResult<User> {
-        let full_url = self.api_base_url.to_string() + "user";
+        let full_url = format!("{}user", self.api_base_url);
```

Comment:
If I am not mistaken "URL//user" equals to "URL/user". So it is more readable to use `format!("{}/user")`. Also it will protect you from incorrect `api_base_url` if it does not contain trailing `'/'`.